src/containers/Breadcrumbs/Breadcrumbs.jsx

```diff
@@ -17,14 +17,35 @@ import {
   setUri,
   setUriChanged,
 } from '/src/features/context'
+import { upperFirst } from 'lodash'
 
-const uri2crumbs = (uri) => {
-  if (!uri) return []
-
+const uri2crumbs = (uri = '', pathname) => {
   // parse uri to path and query params
-
-  const [path, query] = uri.split('://')[1].split('?')
+  const [scope, pathAndQuery = ''] = uri.split('://')
+  const [path, query] = pathAndQuery.split('?')
   const crumbs = path.split('/').filter((crumb) => crumb)
+
+  if (scope?.includes('ayon+settings')) {
```

Comment:
I guess this could also be a switch?

```js
switch (scope) {
  case "ayon+settings":
    let settingsScope = query?.includes('project') ? 'Projects Manager' : 'Studio Settings'
    crumbs.unshift(settingsScope)
    break;
  case "ayon+entity":
    crumbs.unshift("Project")
    break;
  default:
    let pageTitle = pathname.split("/")[1]
    if (pageTitle.includes('settings')) pageTitle = 'Studio Settings'
    else if (pageTitle.includes('manageProjects')) pageTitle = 'Projects Manager'
    // just a regular url
    crumbs.unshift(upperFirst(pageTitle));
}
```

Regardless,  `pathname.split("/")[1]` might fail if `pathname` contains no slashes (unsure if possible)